client/src/components/App.jsx

```diff
@@ -10,50 +10,61 @@ export const App = () => {
   //******************************
   // STATE
   //******************************
-  const [product_id, setProductId] = useState(63609);
-  const [productStyle, setProductStyle] = useState(63609)
-  const [reviewMetaData, setReviewMetaData] = useState();
-  const [reviews, setReviews] = useState();
+  const [productId, setProductId] = useState(63609);
+  const [product, setProduct] = useState({});
```

Comment:
The `product_id` variable was being overwritten by the fetched product data, so I made a separate `product` variable.